bench-tps/src/bench.rs

```diff
@@ -176,7 +168,7 @@ where
     let start = Instant::now();
     let keypair_chunks = source_keypair_chunks.len() as u64;
     let mut reclaim_lamports_back_to_source_account = false;
-    let mut i = keypair0_balance;
+    let mut i = 0;
```

Comment:
why change this?

Reply:
So that mechanism was in place to quickly restart `bench-tps` with the same parameters as a previous run. We would detect the balance of the last keypair to check how many lamports had been transferred from one half of they keypairs to the other half. But I found that the assumptions were not valid for running with `ramp-tps`.

In this PR, I removed the quick-start mechanism but I think that was a mistake. I'll think of another approach that works well for repeated `bench-tps` runs as well as incremental `ramp-tps` runs